src/mbgl/map/transform.cpp

```diff
@@ -517,7 +527,7 @@ double Transform::getAngle() const {
 #pragma mark - Pitch
 
 void Transform::setPitch(double pitch, const Duration& duration) {
-    setPitch(pitch, {NAN, NAN}, duration);
+    setPitch(pitch, ScreenCoordinate {}, duration);
```

Comment:
You are right @1ec5 - I am reverting this particular change in https://github.com/mapbox/mapbox-gl-native/pull/4277.